staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go

```diff
@@ -978,6 +978,22 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 		vmSetName := az.mapLoadBalancerNameToVMSet(lbName, clusterName)
 		// Etag would be changed when updating backend pools, so invalidate lbCache after it.
 		defer az.lbCache.Delete(lbName)
+
+		switch az.PreConfiguredBackendPoolLoadBalancerTypes {
```

Comment:
I thin this is not sufficient, at L936 we also delete the backend pool and delete LB. IMO in the "preconfigured" case we should only configure LB rules, but not managing LB itself and backend pools.

Reply:
make sense. thanks @yangl900

Reply:
related logic is added